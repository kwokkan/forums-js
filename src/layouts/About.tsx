import { Content, Panel } from "rsuite";

export function About() {
    return (
        <Panel header={<h3>About</h3>}>
            <Content>
                Forums JS is a forums app written in JS.
            </Content>
        </Panel>
    );
}
