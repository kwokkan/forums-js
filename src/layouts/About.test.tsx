import { create } from "react-test-renderer";
import { About } from "./About";

test("Renders without errors", () => {
    const tree = create(
        <About />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
