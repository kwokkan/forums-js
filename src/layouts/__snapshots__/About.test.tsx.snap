// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without errors 1`] = `
<div
  className="rs-panel rs-panel-default"
>
  <div
    className="rs-panel-heading"
    onClick={[Function]}
    role="rowheader"
    tabIndex={-1}
  >
    <h3
      className="rs-panel-title"
    >
      About
    </h3>
  </div>
  <div
    className="rs-panel-body"
  >
    <div
      className="rs-content"
    >
      Forums JS is a forums app written in JS.
    </div>
  </div>
</div>
`;
